book/src/feature/logging.md

```diff
@@ -1,3 +1,8 @@
+# Stack Traces
```

Comment:
IMHO it would be great to:

1. move stack traces into separate file, say `feature/stack_trace.md`
2. Add a sentence in this file, saying "if you want to have stack trace, see that file" so users can link to there

Reply:
I was thinking about that, but the section is so tiny, that I thought it better fits into `Logging`, but maybe a section `Debugging` would make more sense.

Reply:
IMHO tiny is not a big issue. Instead, it is important for readers to quickly find them - otherwise it is equivalent to not write down it at all ;)

Reply:
ok done